test-case-reporting/src/setup_db.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Database, dbConnect} from './db';
+
+/** Set up the database table. */
+export async function setupDb(db: Database): Promise<unknown> {
+  return db.schema
+    .createTable('builds', table => {
+      table.increments('id').primary();
+      table.string('commit_hash', 40);
+      table.integer('pr_number');
+      table.dateTime('started_at');
+    })
+    .createTable('jobs', table => {
+      table.increments('id').primary();
+      table.foreign('build_id').references('builds.id');
+      table.string('job_num');
```

Comment:
I'd say we can be slightly more verbose (ie. `'job_number'`) to match `pr_number`

Reply:
Done.